test/frontend/MemberDashboard.test.js

```diff
@@ -0,0 +1,29 @@
+/* global describe it */
+import 'jsdom-global/register'
+import React from 'react'
+import Enzyme, { mount } from 'enzyme'
+import { expect } from 'chai'
+
+import MemberDashboard from '../../src/Pages/Overview/App.js'
+import Adapter from 'enzyme-adapter-react-16'
+
+Enzyme.configure({ adapter: new Adapter() })
+
+describe('<MemmberDashboard />', () => {
+  Object.defineProperty(window, 'localStorage', {
+    value: global.localStorage,
+    configurable: true,
+    enumerable: true,
+    writable: true
+  })
+
+  it('Should render a <table /> component with one child', () => {
+    const wrapper = mount(<MemberDashboard />)
+    expect(wrapper.find('table')).to.have.lengthOf(1)
+  })
+
+  it('Should render a <table /> component with one child', () => {
```

Comment:
Add another test case in here making sure that components will hold text values,

here is an [example](https://github.com/SCE-Development/Core-v4/blob/5965f2e1c00b49278abdf5a9a421f9025f1237d8/test/frontend/2dPrintingTest.js#L24)

Reply:
Can someone write this? I want to focus on getting the meat done

Reply:
your component, your tests man